catalyst/contrib/dl/callbacks/periodic_loader_callback.py

```diff
@@ -12,13 +12,31 @@ class PeriodicLoaderCallback(Callback):
     """Callback for runing loaders with specified period.
     To disable loader use ``0`` as period.
 
-    Example:
+    For example, if you have ``train``, ``train_additional``,
+    ``valid`` and ``valid_additional`` loaders and wan't to
+    use ``train_additional`` every 2 epochs, ``valid`` - every
+    3 epochs and ``valid_additional`` - every 5 epochs:
+
+    .. code-block:: python
+
+        from catalyst.dl import (
+            SupervisedRunner, PeriodicLoaderRunnerCallback,
+        )
+        runner = SupervisedRunner()
+        runner.train(
+            ...
```

Comment:
would you like to add `loaders={"train_additional": ..., "valid": ..., "valid_additional":...}`?

Reply:
yep, i agree, need to add this